### PR TITLE
Fix eslint errors on tests

### PR DESCRIPTION
**Pull request by Ladsgroup**
Still keeping the .eslintrc.json file for qunit

### CODE REVIEW

File: tests/DeserializerFactory.tests.js [hunk 1]
```diff
@@ -21,14 +21,14 @@ QUnit.test( 'registerDeserializer(), newDeserializerFor()', function( assert ) {
 		}
 	];
 
-	assert.throws(
+	assert['throws'](
```
Comment (manicki) [inline tests/DeserializerFactory.tests.js]: As I was trying to say on the other PR. I'd rather ignore dot-notation rule in tests and keep the "old" form here. assert['throws'] looks silly given assert.ok etc are "fine" by eslint.

**Comment by manicki** [comment]
Seems ok but I am not convinced by assert['throws']

**Comment by Ladsgroup** [comment]
I removed the assert['throws'] parts

File: tests/SerializerFactory.tests.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ QUnit.test( 'registerSerializer(), newSerializerFor()', function( assert ) {
 	);
 
 	assert.ok(
-		serializerFactory.newSerializerFor( new( testSets[0].Constructor ) ).constructor
+		serializerFactory.newSerializerFor( new ( testSets[0].Constructor ) ).constructor
```
Comment (thiemowmde) [inline tests/SerializerFactory.tests.js]: The brackets don't do anything here and should be removed, in my opinion.

Also I believe the syntax should be `new testSets[0].Constructor()`.

**Pull request merged**